ika/client.go

```diff
@@ -78,3 +85,86 @@ func (p *Client) UpdateLC(
 		RequestType: "WaitForLocalExecution",
 	})
 }
+
+// ApproveAndSign approves and signs a set of messages using the IKA network. Returns its signatures
+func (p *Client) ApproveAndSign(
+	ctx context.Context,
+	dwalletCapID string,
+	signMessagesID string,
+	messages [][]byte,
+) ([][]byte, error) {
+
+	// TODO: This function was only tested against dummy implementation of the dwallet module deployed locally.
+	// Once it is ready, test it again
+	req := models.MoveCallRequest{
+		Signer:          p.Signer.Address,
+		PackageObjectId: p.LcPackage,
```

Comment:
This is not LcPackage. I think the correct one is dWalletPackage.

Reply:
Do you think we should extend the `Client` to include those?

Reply:
Pre-approved!

Reply:
Done